### DISCUSSION TITLE
Intro to Github flow can't mark complete

### DISCUSSION THREAD

**Discussion by lilimu996** [community/community#118763] [GitHub Education]
### Hi everyone,

I'm using the intro to github flow from learn new skills.and I confirm that I've followed the steps in the tutorial to complete this task.
![image](https://github.com/community/community/assets/53130598/706d24ca-4fb6-420d-812f-c19fc65c4031)
But I can't mark this task as complete，I'll get a error that We were unable to find a Classroom assignment for your account.
![image](https://github.com/community/community/assets/53130598/7921d530-8f11-4a26-8586-6fedc283b27c)
Then I tried to fork this repository to my own account，I did some more updates in the fork repository and then merged it into the original repository,But it's still the same error.
So I'm missing something?

**Comment by LeanderFischer** [comment]
Same happening to me, haven't solved it, either..

**Comment by BilalHaider20** [comment]
I am facing this same issue. Have not found solution either.

**Comment by rudychan000** [comment]
Same problem, the only thing I find about this problem is this post. 😢

**Comment by Flavio13Lins** [comment]
Same to me, saw some people that tried to hack it making the open tab as "X".

**Comment by ethan548** [comment]
Same to me...

**Comment by Flavio13Lins** [comment]
For documentation propose:
- [GitHub Education Experience](https://education.github.com/experiences/series_communicate_using_markdown)
- [GitHub Education Repo](https://github.com/Exp-Communicate-Using-Markdown-Cohort-1)

@lilimu996, did you merged the content from 'dependabot' PRs before starting the lab?
Because I did it and I don't know if it may cause some problem.

[My Repo](https://github.com/Exp-Communicate-Using-Markdown-Cohort-1/series-communicate-using-markdown-Flavio13Lins)

**Reply by lilimu996** [reply]
I didn't do that. I followed the lab steps exactly.

**Comment by lilimu996** [comment]
I've stopped trying to solve this problem, not marking this assignments as complete doesn't seem to affect the learning of other assignments.You can see all the assignments in Explore More Offers.
![image](https://github.com/community/community/assets/53130598/49c14658-db34-4653-b568-4e30087b702c)

**Comment by jialianghua** [comment]
Hi, same problem, happened today. I hope Github can resolve this....

**Comment by jinkajuice** [comment]
Same problem here.
